conf/messages

```diff
@@ -280,6 +281,16 @@ createAccount.group.details.header.multipleAddress=There’s more than one addre
 createAccount.group.details.content.multipleAddress=You should enter the address that you want any business correspondence to be sent to.
 createAccount.business.email.help=This should be your email address and not your agent’s. The email address should be monitored.
 
+registration.successful=Registration successful
+voa.personal.id.number=Your VOA personal ID number
+agent.code=Your Agent code
+email.sent.to=We have sent these details to <strong>{0}</strong>
+next.time.sign.in=Next time you sign in to the service, use your Government Gateway details.
+reset.password=If you need to reset your password, you'll need to provide your VOA personal ID number.
+give.agent.code.to.clients=Give your Agent code to your client so they can appoint you to act for them.
```

Comment:
the message keys are not what we usually do, i think... in `give.agent.code.to.clients`, the text could be later changed to say something totally different, e.g. `give.agent.code.to.clients=Contact the VOA for a free valuation` and then they the key will make no sense. I think we used hierarchy of where the text sits on the page, like `pageName.sectionName.p1` ... I am not too bothered about this, but I wonder what others think

Reply:
let's get it right 🙂
key must start with `pageName` at least - after that it gets tricky, but the key should be based on function or page-structure (as pete said), not on content of the message

Reply:
keys updated